webapp/components/post_view/post_body_additional_content.jsx

```diff
@@ -8,6 +8,7 @@ import YoutubeVideo from 'components/youtube_video';
 
 import Constants from 'utils/constants.jsx';
 import * as Utils from 'utils/utils.jsx';
+import BrowserStore from '../../stores/browser_store.jsx';
```

Comment:
Can you remove the relative path and just `stores/browser_store.jsx`? Saves us some pain if this file moves around

Reply:
I have made the changes. Have a look now